db/seeds.rb

```diff
@@ -5,8 +5,7 @@
 [
   { name: 'Factual', rank: 1 },
   { name: 'HumanOutsourcer', rank: 2 },
-  { name: 'Geocoder', rank: 3 },
-  { name: 'Burden', rank: 4 }
+  { name: 'Burden', rank: 3 }
```

Comment:
If we do this, we should refactor the Geocoder: https://github.com/artsy/bearden/blob/fb45e09545525aeadabdb95bbe45010128378e04/lib/tasks/geocode.rake#L3